src/components/OurServicesSection/ourservices.css

```diff
@@ -0,0 +1,80 @@
+.ourServicesContainer {
+  width: 100%;
+  background-color: #f6f8f9;
+  display: flex;
+  justify-content: center;
+}
+.ourServicesWrapper {
+  width: 80%;
+  padding: 4.625rem 3.90625%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: wrap;
+}
+@media (max-width: 768px) {
+  .ourServicesWrapper {
+    display: flex;
+    flex-direction: column;
+  }
+}
+.servicesHeader {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 1.25rem;
+}
+.serviceOffers {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+@media (max-width: 768px) {
```

Comment:
Same here for the media query. You can use only one max-width 768px to achieve your goal in this file instead of duplicating it.